src/lib/isAfter.js

```diff
@@ -1,8 +1,13 @@
 import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isAfter(str, date = String(new Date())) {
+export default function isAfter(str, options) {
   assertString(str);
+
+  // accessing 'arguments' for backwards compatibility: isAfter(str [, date])
+  // eslint-disable-next-line prefer-rest-params
+  const date = (typeof options === 'object' ? options.date : arguments[1]) || String(new Date());
```

Comment:
suggestion: I'd think the idiomatic way to do this is Date().toString() :).

Reply:
Updated, thanks!